docs/pages/project/releases.md

```diff
@@ -5,7 +5,7 @@ permalink: project/releases
 abstract: List of released Meshery versions and their release notes.
 redirect_from: project/releases/
 language: en
-type: project
+type: Reference
```

Comment:
type=project is being used on homepage to make the list: https://github.com/meshery/meshery/blob/fe72c24fd686ce53b739daf81351c59bee385493/docs/pages/index.md?plain=1#L347

Reply:
oh i see, let me see how we can fix this